ftw/showroom/js/src/ftw.showroom.js

```diff
@@ -107,7 +107,9 @@ module.exports = function Showroom(items = [], options) {
         element.show();
         target.append(element).addClass("ftw-showroom-open");
         checkArrows();
-    });
+      }).done(() => {
```

Comment:
Use `pipe` instead of `done`. I guess the `done` breaks the pipe and it's no longer possible to extend the pipe after the `done`.

Reply:
Updated 👍